src/lib/yang/parser.lua

```diff
@@ -282,8 +288,13 @@ function Parser:parse_statement()
 
    -- Take the identifier
    self:consume_whitespace()
-   local argument = self:parse_string()
```

Comment:
I want to avoid changing `parse_string` here.  I think what should happen instead is that here at this line we need to do a look-ahead.  How about instead:

```lua
if self:peek() ~= ';' and self:peek() ~= '{' then
  returnval.argument = self:parse_string()
end
```

Then everything else is the same.  WDYT?